### PR TITLE
IOS 495 - “colorPickerIcon” asset is missing. It needs to be added from the design docs

### PR DESCRIPTION
**Pull request by an-nguyen-d**
### Jira Ticket
<Adjust the ticket below to match the ticket>
[IOS 495 - “colorPickerIcon” asset is missing. It needs to be added from the design docs](https://unumdesign.atlassian.net/browse/IOS-495)

### Purpose
In the description of the ticket it's mentioned that the image picker is too far down. After looking into it, it seems like this will be fixed when the color picker buttons matches the specs in the sketch files: https://files.slack.com/files-pri/T9YHBJQ2U-FJRST73SA/download/final_mobile___grid_offset.sketch

#### Before
IMG_4099

#### After
IMG_4098

### CODE REVIEW

**Comment by an-nguyen-d** [comment]
Not sure what all the pod changes are, all the code change was basically in the storyboard file and the collection cell

File: Example/Podfile.lock [hunk 1]
```diff
@@ -9,8 +9,8 @@ EXTERNAL SOURCES:
     :path: "../"
 
 SPEC CHECKSUMS:
-  UNUMColorPicker: 97d69496071ef08cc9faa8f6ef80e1af7e6562b5
+  UNUMColorPicker: bb844712b3dcfcf833941726fcb51ebb7e319e10
 
 PODFILE CHECKSUM: b91a5bcb60dc34caf8b8a315bc094f034ed944ab
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.6.1
```
Comment (plflanagan) [inline Example/Podfile.lock:16]: Hmm, aren't you using 1.6.2?

**Comment by plflanagan** [comment]
I'm not sure what the proper treatment for the save/cancel buttons are; so bringing in design to clarify.

**Comment by plflanagan** [comment]
Based on this conversation we should keep the save and cancel buttons:

https://teamunum.slack.com/archives/CE8RVJ54Z/p1558037391003300

**Pull request closed without merging**